index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link href="style.css" rel="stylesheet" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <header id="header">
+            <img id="header-img" src="logo.png"/>
+            <nav id="nav-bar">
+                <a class="nav-link" href="#features">Features</a>
+                <a class="nav-link" href="#video">How It Works</a>
+                <a class="nav-link" href="#prices">Pricing</a>
+            </nav>
+        </header>
+        <div id="content">
+            <h2>Handcrafted, home-made masterpieces</h2>
+            <form id="form" action="https://www.freecodecamp.com/email-submit">
+                <input type="email" id="email" placeholder="Enter your email address"/>
+                <button type="submit" id="submit">Get Started</button>
+            </form>
+            <div id="features">
+                <div>
+                    <span>Premium Materials</span>
+                    <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.</p>
+                </div>
+                <div>
+                    <span>Fast Shipping</span>
+                    <p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.</p>
+                </div>
+                <div>
+                    <span>Quality Assurance</span>
+                    <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.</p>
+                </div>
+            </div>
+            <iframe id="video"
+                src="https://www.youtube-nocookie.com/embed/y8Yv4pnO7qc" 
+                title="YouTube video player" frameborder="0" 
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" 
+                allowfullscreen>
+            </iframe>
+            <div id="prices">
```

Comment:
- [ ] _( optional )_ I think it would be better to use "< section >" tag instead of < div > for your code to be more semantic